README.md

```diff
@@ -294,9 +294,7 @@ package manager for Mac OS X.
 
 To upgrade pyenv in the future, use `upgrade` instead of `install`.
 
-After installation, you'll need to add `eval "$(pyenv init -)"` to your profile (as stated in the caveats displayed by Homebrew — to display them again, use `brew info pyenv`). You only need to add that to your profile once.
-
-Then follow the rest of the post-installation steps under "Basic GitHub Checkout" above, starting with #4 ("restart your shell so the path changes take effect").
+Then follow the rest of the post-installation steps under "Basic GitHub Checkout" above, starting with #3 ("Add `pyenv init` to your shell to enable shims and autocompletion").
```

Comment:
You can make the "Basic GitHub Checkout" into a link to the appropriate section: https://github.com/pyenv/pyenv#basic-github-checkout